orodha-keycloak/tests/test_operations.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+import orodha_keycloak.connection
+from tests.fixtures.keycloak import MOCK_DATA
+
```

Comment:
@Edison-Stuart Here, you're testing `orodha_keycloak.connection`, so you might consider naming the test file after the files it's testing. E.g., `test_connection.py` as opposed to `test_operations.py`. This may seem trivial now, with a small amount of code & tests, but with a larger codebase (potentially being maintained by many devs), having the test organized by the code they're testing is quite helpful.

Reply:
Noted, I've changed the name of the file

Reply:
@Edison-Stuart The file still looks like `test_operations.py`

Reply:
When I look at the current files changed I see test_connection.py. However I have to remove the keycloak directory that is mounted as a volume from the user service as well as add /dist to the gitignore, so I will take another look at this too and then squash some of the commits down as well.